example.js

```diff
@@ -5,7 +5,8 @@ var password = generator.generate({
 	length: 15, // defaults to 10
 	numbers: true, // defaults to false
 	symbols: true, // defaults to false
-	uppercase: true // defaults to true
+	uppercase: true, // defaults to true
+	strict : true // defaults to false - it will always have at least 1 lowercase letter
```

Comment:
Maybe say it'll have at least one symbol and number instead? Also, you have an extra space before the colon.